modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/portlet_url.js

```diff
@@ -240,43 +238,20 @@ AUI.add(
 			toString: function() {
 				var instance = this;
 
-				var options = instance.options;
```

Comment:
We should still use `var options = instance.options;`, replacing in later lines.